cmds/core/unshare/unshare_linux.go

```diff
@@ -43,49 +43,48 @@ import (
 	"syscall"
 )
 
-var (
-	ipc   = flag.Bool("ipc", false, "Unshare the IPC namespace")
-	mount = flag.Bool("mount", false, "Unshare the mount namespace")
-	pid   = flag.Bool("pid", false, "Unshare the pid namespace")
-	net   = flag.Bool("net", false, "Unshare the net namespace")
-	uts   = flag.Bool("uts", false, "Unshare the uts namespace")
-	user  = flag.Bool("user", false, "Unshare the user namespace")
-)
-
-func main() {
-	flag.Parse()
-
-	a := flag.Args()
-	if len(a) == 0 {
-		a = []string{"/bin/sh"}
+func command(ipc, mount, pid, net, uts, user bool, args []string) *exec.Cmd {
```

Comment:
should args be ...string, since it's allowed to be empty? 
Should we be using the func (c*cmd) Command style here?